packages/manager/modules/pci/src/projects/project/storages/databases/add/add.constants.js

```diff
@@ -1,9 +1,373 @@
+import { random } from 'lodash';
+
 export const NAME_PATTERN = /^([\w.-]*)$/;
 export const MIN_NAME_LENGTH = 3;
 export const MAX_NAME_LENGTH = 40;
+export const ORDER_KEYS = [
+  'description',
+  'nodesPattern.flavor',
+  'nodesPattern.number',
+  'nodesPattern.region',
+  'plan',
+  'version',
+  'networkId',
+  'subnetId',
+];
+
+const adjectives = [
```

Comment:
PU could expose an API endpoint to get those values ?

Reply:
I used the same system that was made for data-processing: https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/projects/project/data-processing/data-processing.utils.js#L88
I have talked with PU about exposing it in a API, and for now it is not possible as there is no transversal api yet.

Reply:
Even without putting it in an API, maybe we could have it in a utility service common to the PU ?

Reply:
Moved: https://github.com/ovh/manager/pull/6905/commits/ec6d484d1576225ef6708192fbda66b974b3171d